v7/recent_posts_json/recent_posts_json.py

```diff
@@ -131,7 +132,10 @@ def make_json(self, posts, descriptions, previewimage, output_path, lang):
             recent_posts.append(entry)
         data = json.dumps(recent_posts, indent=2, sort_keys=True)
         with io.open(output_path, "w+", encoding="utf8") as outf:
-            outf.write(data)
+            if sys.version_info[0] != 3:
```

Comment:
yea, excepting more thing makes it more clear.

We don't decode random unicode, we decode output from json.dumps:
    
    >>> import json
    >>> json.dumps([u"ą"]).decode('utf-8')
    u'["\\u0105"]'
    >>>